go/tools/testwrapper/wrap.go

```diff
@@ -0,0 +1,31 @@
+package testwrapper
+
+import (
+	"bytes"
+	"io"
+	"os"
+
+	"os/exec"
+)
+
+// WrapIfNecessary wraps the execution of the test
+func WrapIfNecessary() {
+	if os.Getenv("RAN_FROM_BAZEL") != "1" {
+		return
+	}
+	os.Unsetenv("RAN_FROM_BAZEL")
+
+	exitCode := 0
```

Comment:
Let's pick a distinctive value for an abnormal termination. Not sure if there's a standard code, but Bazel uses 6 to mean `RUN_FAILURE` in [ExitCodes.java](https://github.com/bazelbuild/bazel/blob/master/src/main/java/com/google/devtools/build/lib/util/ExitCode.java), so I suggested that above.